src/cobalt.rs

```diff
@@ -190,7 +190,7 @@ pub fn build(config: &Config) -> Result<()> {
         }
 
         let mut context = doc.get_render_context(&posts_data);
-        let doc_html = try!(doc.render(&mut context, source, &layouts, &mut layouts_cache));
+        let doc_html = try!(doc.render(&mut context, &source, &layouts, &mut layouts_cache));
```

Comment:
This change is squashed across two commits.

Bad revert on my part.  @mre had appended a `&config` parameter and switched `source` to a reference.  `&config` wasn't necessary if using a global variable to store the theme name, and I failed to restore `&source` to `source`.

If we go with the alternate patch set, then the reversion doesn't happen.  If you prefer to keep `source` over `&source`, I'll need to update that patch set.